### PR TITLE
[recipe, doc] feat: update full-weight Qwen-Image OCR FlowGRPO recipe

### PR DESCRIPTION
**Pull request by AndyZhou952**
### What does this PR do?

> Add **concise** overview of what this PR aims to achieve or accomplish. Reference related GitHub issues and PRs that help with the review.

- update the full-weight training script: increased lr (1e-5 -> 3e-5) for faster convergence and used 512x512 training
- convergence in ~90 steps (vs. 210 steps with previous script
- update relevant performance doc

### Checklist Before Starting

- [ ] Search for similar PRs. Paste at least one query link here: ...
- [ ] Format the PR title as `[{modules}] {type}: {description}` (This will be checked by the CI)
  - `{modules}` include `fsdp`, `vllm_omni`, `rollout`, `trainer`, `ci`, `training_utils`, `recipe`, `ray`, `worker`, `single_controller`, `misc`, `perf`, `model`, `algo`, `env`, `tool`, `ckpt`, `doc`, `data`, `cfg`, `reward`, `diffusion`, `omni`, `tests`, `docker`
  - If this PR involves multiple modules, separate them with `,` like `[diffusion, doc]`
  - `{type}` is in `feat`, `fix`, `refactor`, `chore`, `test`
  - If this PR breaks any API (CLI arguments, config, function signature, etc.), add `[BREAKING]` to the beginning of the title.
  - Example: `[BREAKING][diffusion, fsdp] feat: new rollout scheduler`

### Test

> For changes that can not be tested by CI (e.g., algorithm implementation, new model support), validate by experiment(s) and show results like training curve plots, evaluation results, etc.

Experiments are performed on 4 NVIDIA H200s w/ no cfg (i.e., true_cfg_config=1.0, set optimizer buffer to fp32, image resolution 512x512, lr=3e-5, clip range=1e-5). 

image
image
image

### API and Usage Example

> Demonstrate how the API changes if any, and provide usage example(s) if possible.

```python
# Add code snippet or script demonstrating how to use this
```

### Design & Code Changes

> Demonstrate the high-level design if this PR is complex, and list the specific changes.

### Checklist Before Submitting

> [!IMPORTANT]
> Please check all the following items before requesting a review, otherwise the reviewer might deprioritize this PR for review.

- [ ] Read the [Contribute Guide](https://github.com/verl-project/verl-omni/blob/main/CONTRIBUTING.md).
- [ ] Apply [pre-commit checks](https://github.com/verl-project/verl-omni/blob/main/CONTRIBUTING.md#code-linting-and-formatting): `pre-commit install && pre-commit run --all-files --show-diff-on-failure --color=always`
- [ ] Add / Update the documentation.
- [ ] Add unit or end-to-end test(s) to [the CI workflow](https://github.com/verl-project/verl-omni/tree/main/.github/workflows) to cover all the code. If not feasible, explain why: ...

### CODE REVIEW

**Review by zhtmike — APPROVED** [review]

File: docs/algo/performance.md [hunk 1]
```diff
@@ -40,24 +40,30 @@ Evaluated with `trainer.val_before_train=True`:
 
 ## FlowGRPO: non-CFG Full Model Training on Qwen-Image OCR
 
-> Experiments used NVIDIA H200 GPUs, `ppo_micro_batch_size_per_gpu` 8, lr 1e-5, clip_ratio 1e-5, image resolution 384x384, optimizer state fp32. The other parameters are consistent with the LoRA setting.
+> Experiments used NVIDIA H200 GPUs, lr 3e-5, clip_ratio 1e-5, optimizer state fp32. The other parameters are consistent with the LoRA setting.
```
[the remark this answers is not in the document]
Comment (AndyZhou952) [reply docs/algo/performance.md:43]: no need. 512x512 is consistent with the lora setting.

**Review by SamitHuang — COMMENTED** [review]
LGTM

**Review by SamitHuang — APPROVED** [review]
